content/3.workflow/2.BNO-onboarding.md

```diff
@@ -6,3 +6,12 @@ The BNO onboarding process consists of the following steps:
 2. The BNO provides HTTPS endpoint details to the PSP.
 3. The BNO sets up the Payments Gateway as a node within their Corda network (Gevamu Payments Gateway binaries are provided by Exactpro) using configuration details exchanged previously.
 
+```mermaid
+
+sequenceDiagram
+
+    PSP ->> BNO: 1
+    BNO ->> PSP: 2
+    BNO ->> GevamuPaymentGateway: 3
+
+```
```

Comment:
![image](https://github.com/gevamu/gevamu.github.io/assets/51794590/ceee6e02-693c-426c-8e7c-a55d03a81036)

Check `autonumber` option in mermaid. It looks better.
Provide most valuable information about actions directly on diagram. No it is just links to some list item. It is hard to read.

Reply:
Kolya the idea was that the numbers in the sequence diagram correspond with the list above. Since each list item has a detailed explanation I thought it would be best to refer it. what do you think ?